FBAllocationTracker/NSObject+FBAllocationTracker.h

```diff
@@ -31,3 +31,17 @@
 @end
 
 #endif // _INTERNAL_FBAT_ENABLED
+
+/**
+ Returns whether `object` is the zombie object.
+ 
+ @return `true` if `object` is nil.
+ @return `false` if `object` is not nil and NSZombieEnabled = NO.
+ */
+bool fb_isZombieObject(_Nullable id object);
```

Comment:
Sorry for being picky, but what do you think about extracting these declarations and implementation to other files? Like FBAllocationTrackerZombieSupport, or something else that comes to mind? I'm not strong on that, but I feel it might make context easier to understand.